### DISCUSSION TITLE
clientError generated after all persistent connection messages are deliverd

### DISCUSSION THREAD

**Discussion by aqibamir** [moscajs/aedes#811] [Q&A]
Whenver a client receives all the messags that were generated when the client was offline it abruptly dies by generating the following clientError : no such packet

<img width="973" alt="Screenshot 2023-02-21 at 1 29 45 AM" src="https://user-images.githubusercontent.com/73932489/220193329-af81608e-33a0-4074-83ca-c4750ac1ee80.png">

What might be causing this ? I'm using all default options in my server

**Comment by robertsLando** [comment]
If possible try to see if the problem persists even after cleaning the database. aedes-persistence-redis 9.x.x is not fully compatible with 8.x.x, see https://github.com/moscajs/aedes-persistence-redis/blob/master/UPGRADE.md

**Reply by aqibamir** [reply]
I'm using local persistence. Redis warning is being generated by another operation.

**Reply by robertsLando** [reply]
Are you using 0.48.1? Are you able to debug this down to the line of code that causes the problem? Feel free to send a PR
